content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -0,0 +1,271 @@
+
+### 네트워크 구성
+
+리눅스 기반의 쿠버네티스 마스터 노드를 가지고 있다면 네트워킹 솔루션을 선택할 준비가 된 것이다. 이 가이드는 단순화를 위해 VXLAN 방식의 플라넬(Flannel)을 사용하여 설명한다.
+
+#### 리눅스 컨트롤러에서 VXLAN 방식으로 플라넬 구성하기
+
+1. 플라넬을 위해 쿠버네티스 마스터를 준비한다.
+
+    클러스터의 쿠버네티스 마스터에서 사소한 준비를 권장한다. 플라넬을 사용할 때에 iptables 체인으로 IPv4 트래픽을 브릿지할 수 있게 하는 것은 추천한다. 이는 다음 명령어를 이용하여 수행할 수 있다.
+
+    ```bash
+    sudo sysctl net.bridge.bridge-nf-call-iptables=1
+    ```
+
+1. 플라넬 다운로드 받고 구성하기
+
+    가장 최신의 플라넬 메니페스트를 다운로드한다.
+
+    ```bash
+    wget https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
+    ```
+
+    VXLAN 네트워킹 벡엔드를 가능하게 하기 위해 수정할 곳은 두 곳이다.
+
+    아래 단계를 적용하면 `kube-flannel.yml`의 `net-conf.json`부분을 다음과 같게 된다.
+
+    ```json
+    net-conf.json: |
+        {
+          "Network": "10.244.0.0/16",
+          "Backend": {
+            "Type": "vxlan",
+            "VNI" : 4096,
+            "Port": 4789
+          }
+        }
+    ```
+
+    {{< note >}}리눅스의 플라넬과 윈도우의 플라넬이 상호운용하기 위해서 `VNI`는 반드시 4096이고, `Port`는 4789여야 한다. 다른 VNI는 곧 지원한다. [VXLAN 문서](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan)에서
+     이 필드의 설명 부분을 보자.{{< /note >}}
+
+1. `kube-flannel.yml`의 `net-conf.json` 부분을 거듭 확인하자.
+    1. 클러스터 서브넷(예, "10.244.0.0/16")는 IP 주소 설계에 따라 설정되어야 한다.
+        * VNI 4096 은 벡엔드에 설정한다.
+        * Port 4789 은 벡엔드에 설정한다.
+    2. `kube-flannel.yml`의 `cni-conf.json` 부분에서 네트워크 이름을 `vxlan0`로 바꾼다.
+
+
+    `cni-conf.json`는 다음과 같다.
+
+    ```json
+    cni-conf.json: |
+        {
+          "name": "vxlan0",
+          "plugins": [
+            {
+              "type": "flannel",
+              "delegate": {
+                "hairpinMode": true,
+                "isDefaultGateway": true
+              }
+            },
+            {
+              "type": "portmap",
+              "capabilities": {
+                "portMappings": true
+              }
+            }
+          ]
+        }
+    ```
+
+1. 플라넬 yaml 을 적용하고 확인하기
+
+    플라넬 구성을 적용하자.
+
+    ```bash
+    kubectl apply -f kube-flannel.yml
+    ```
+
+    다음은 플라넬 파드는 리눅스 기반이라 [여기](https://github.com/Microsoft/SDN/blob/1d5c055bb195fecba07ad094d2d7c18c188f9d2d/Kubernetes/flannel/l2bridge/manifests/node-selector-patch.yml)나온 `NodeSelector` 패치를 플라넬 데몬셋 파드에 적용한다.
+
+    ```bash
+    kubectl patch ds/kube-flannel-ds-amd64 --patch "$(cat node-selector-patch.yml)" -n=kube-system
+    ```
+
+    몇 분 뒤에 플라넬 파드 네트워크가 배포되었다면, 모든 파드에서 운영 중인 것을 확인할 수 있다.
+
+    ```bash
+    kubectl get pods --all-namespaces
+    ```
+
+    ![alt_text](../flannel-master-kubectl-get-pods.png "플라넬 마스터에서 kubectl get pods 스크린 캡춰")
+
+    플라넬 데몬셋에 `NodeSelector`가 적용되었음을 확인한다.
+
+    ```bash
+    kubectl get ds -n kube-system
+    ```
+
+    ![alt_text](../flannel-master-kubectl-get-ds.png "플라넬 마스터에서 kubectl get ds 스크린 캡춰")
+
+#### 윈도우 워커 노드 가입
+
+이번 단원은 맨 땅에서부터 온프레미스 클러스터에 가입하기까지 윈도우 노드 구성을 다룬다. 클러스터가 클라우드상에 있다면, 다음 단원에 있는 클라우드에 특정한 가이드를 따르도록 된다.
+
+#### 윈도우 노드 준비하기
+{{< note >}}
+윈도우 단원에서 모든 코드 부분은 높은(Admin) 권한으로 파워쉘(PowerShell) 환경에서 구동한다.
```

Comment:
```suggestion
윈도우 단원에서 모든 코드 부분은 높은 권한(Admin)으로 파워쉘(PowerShell) 환경에서 구동한다.
```
All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Admin).